client/dashboard/utils/site-provider.ts

```diff
@@ -1,17 +1,14 @@
 import type { Site } from '@automattic/api-core';
 
 export const DEFAULT_PROVIDER_NAME = 'WordPress.com';
-
 export function getSiteProviderName( site: Pick< Site, 'hosting_provider_guess' > ) {
-	const provider = site.hosting_provider_guess;
-	let providerName;
-	if ( provider === 'jurassic_ninja' ) {
-		providerName = 'Jurassic Ninja';
-	} else if ( provider === 'pressable' ) {
-		providerName = 'Pressable';
-	} else if ( provider === 'automattic' ) {
-		providerName = 'WordPress.com';
-	}
+	const providerNameMap: Record< string, string > = {
+		automattic: 'WordPress.com',
```

Comment:
This `automattic => WordPress.com` mapping is sorta problematic for sorting 🤔 

Luckily most provider IDs have the same alphabetical order as their visible label. So when ES sorts them, the labels we give them on the front end won't mess up that order.

But the `automattic` provider is sorted in entirely the wrong direction! If the user sorts the column by ascending then `WordPress.com` will weirdly appear at the top 😳

Reply:
hmm...yes! But I'm not sure how to resolve it 😂